VideoIntelligence/owlbot.py

```diff
@@ -94,6 +94,26 @@
     r"\1](https://cloud.google.com\2)"
 )
 
+# fix phpdoc examples for optional-to-required parameters
+f = open("src/V1/Gapic/VideoIntelligenceServiceGapicClient.php",  "r")
+if ' *     $operationResponse = $videoIntelligenceServiceClient->annotateVideo();' in f.read():
```

Comment:
Should we be adding exceptions like this?
Seems very rigid. One extra space b/w the `*` and `$` and this won't match, right?
Also, if there is an added parameter in the method call, then this won't match if I understand correctly?

Reply:
So, you have to see the PR mentioned in the description to see how this will work. Essentially, it's adding back the PHPdoc that was there previously, but was erased by BC-breaking changes in the protos.

There are two because one example exists at the class level, and the other at the method level. So two different levels of indentation are required. The regex matches the start of the line, so only two lines will be matched by each one (for a total of 4, as you mentioned)

Reply:
It's not beautiful, but it is a pretty nice stop gap until we can come up with something more flexible. I'm okay with this (we've done it in the past). We should definitely make sure we make note of things like this in our playbook, though - so folks know why it is here and what to expect if as @saranshdhingra mentions the regex starts to fail.